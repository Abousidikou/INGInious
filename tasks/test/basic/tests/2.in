{"input":{"q1":"Vagons", "q2":"Ford Prefect"}}
