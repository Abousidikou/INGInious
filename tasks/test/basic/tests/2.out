{"result":"failed", "tests":{"test1":"42"}}
